dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/deletefiles/DeleteFilesPresenter.java

```diff
@@ -52,10 +53,18 @@ public void initialize(final URL location, final ResourceBundle resources) {
         try {
             deleteButton.setDisable(true);
             if (ds3PanelPresenter.getDs3TreeTableView() != null) {
-                final ObservableList<TreeItem<Ds3TreeTableValue>> selectedPanelItems = ds3PanelPresenter.getDs3TreeTableView().getSelectionModel().getSelectedItems();
+                ObservableList<TreeItem<Ds3TreeTableValue>> selectedPanelItems = ds3PanelPresenter.getDs3TreeTableView().getSelectionModel().getSelectedItems();
+                if (null == selectedPanelItems || selectedPanelItems.size() == 0) {
```

Comment:
I believe we have some `Guard.isNullOrEmpty` methods which can be used to perform this check